packages/core/backend/sirius-components-collaborative/src/main/resources/schema/core.graphqls

```diff
@@ -16,7 +16,7 @@ type EditingContext {
   domains: [Domain!]!
   representation(representationId: ID!): RepresentationMetadata
   representations: EditingContextRepresentationConnection!
-  representationDescriptions(kind: ID!): EditingContextRepresentationDescriptionConnection!
+  representationCreationDescriptions(objectId: ID!): EditingContextRepresentationDescriptionConnection!
```

Comment:
Not sure why the name was changed?

Reply:
After discussing with @AxelRICHARD, this was voluntaty, I'll put it back (and update the changelog and adapt the data fetcher in Sirius Web).